src/numgraph/__init__.py

```diff
@@ -1 +1 @@
-from .distributions import barabasi_albert, clique, erdos_renyi, full_grid, random_tree, simple_grid, star, stochastic_block_model
+# from .distributions import barabasi_albert, clique, erdos_renyi, full_grid, random_tree, simple_grid, star, stochastic_block_model
```

Comment:
This will prevent the user from importing like `from numgraph import erdos_renyi_full`. It's not a problem now, but maybe something to think about in the future.